lib/api/resources/mediators.rb

```diff
@@ -15,7 +15,7 @@ class Mediators < Grape::API
       end
 
       get ':id' do
-        mediators = Models::Mediator.where(id: params[:id])
+        mediators = Models::Mediator.where(urn_prefix: params[:id])
```

Comment:
Could you avoid changes to the database by searching on the URN field with the suffix first removed? Something like:

```ruby
get ':urn' do
  urn = params[:urn]
  urn_without_suffix = urn.scan(/\d+/).first
  meditator = Models::Mediator.where(urn: urn_without_suffix)
end
```

Probably being naive about something.